docs/docs/frigate/installation.md

```diff
@@ -243,17 +243,29 @@ you can run Frigate NVR on DSM 7.x. tested on DSM 7.1.1-42962 Update 4
 
 
 General:
+you need to tick the "Execute container using high privilage" checkbox in order to give the frigate container the elevated privilages it may need.
+if you want the container to automatically restart whenever it improperly shuts down then you can tick the "Enable auto-restart" checkbox.
```

Comment:
```suggestion
General:

The `Execute container using high privilage` option needs to be enabled in order to give the frigate container the elevated privileges it may need.

if you want the container to automatically restart whenever it improperly shuts, enable the `Enable auto-restart` option.
```